administrator/components/com_languages/tmpl/language/edit.php

```diff
@@ -9,26 +9,16 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+
 JHtml::addIncludePath(JPATH_COMPONENT . '/helpers/html');
 
 JHtml::_('behavior.formvalidator');
 JHtml::_('behavior.tabstate');
 
-JFactory::getDocument()->addScriptDeclaration(
-	'
-	jQuery(document).ready(function() {
-		jQuery("#jform_image").on("change", function() {
-			var flag = this.value;
-			if (flag) {
-				jQuery("#flag img").attr("src", "' . JUri::root(true) . '" + "/media/mod_languages/images/" + flag + ".gif").attr("alt", flag);
-			}
-			else
-			{
-				jQuery("#flag img").removeAttr("src").removeAttr("alt");
-			}
-	});
-});'
-);
+Factory::getDocument()->addScriptOptions('juri_root', JUri::root(true));
```

Comment:
Same as in #89 : when we use scriptOptions extensively, sooner or later we will have conflicts. 
In this case i would probably prepare the list on php side, so that it returns the full path as value of each option (flag).

Reply:
I have now saved the path as an attribute in the ID flag. So I do not need to use scriptOptions.

Can I specify the value of an option in the field of type filelist? I think it is automatically set equal to the key, right?

Reply:
we can also use https://github.com/joomla/joomla-cms/blob/4.0-dev/libraries/cms/html/behavior.php#L69